test/e2e/functional/tests/e2e.go

```diff
@@ -156,6 +157,8 @@ var _ = Describe("metallb", func() {
 	})
 
 	Context("Creating AddressPool", func() {
+		BeforeEach(createMetalLBResource)
```

Comment:
If you need this to be done in all the tests, you can have BeforeEach / AfterEach in the outer layer

Reply:
ah! yes, starting using it now at required tests.